tests/MartinGeorgiev/Doctrine/ORM/Query/AST/Functions/ToTsqueryTest.php

```diff
@@ -20,13 +20,21 @@ protected function getExpectedSqlStatements(): array
     {
         return [
             'SELECT to_tsquery(c0_.text) AS sclr_0 FROM ContainsText c0_',
+            'SELECT to_tsquery(UPPER(c0_.text)) AS sclr_0 FROM ContainsText c0_',
+            'SELECT to_tsquery(1 + 1) AS sclr_0 FROM ContainsText c0_',
+            'SELECT to_tsquery(1) AS sclr_0 FROM ContainsText c0_',
+            'SELECT to_tsquery(LENGTH(c0_.text)) AS sclr_0 FROM ContainsText c0_',
         ];
     }
 
     protected function getDqlStatements(): array
     {
         return [
             \sprintf('SELECT TO_TSQUERY(e.text) FROM %s e', ContainsText::class),
+            \sprintf('SELECT TO_TSQUERY(UPPER(e.text)) FROM %s e', ContainsText::class),
+            \sprintf('SELECT TO_TSQUERY(1+1) FROM %s e', ContainsText::class),
+            \sprintf('SELECT TO_TSQUERY(true) FROM %s e', ContainsText::class),
+            \sprintf('SELECT TO_TSQUERY(LENGTH(e.text)) FROM %s e', ContainsText::class),
```

Comment:
The checks from lines 35, 36 and 37 won't produce meaningful SQL, as to_tsquery(integer) is not a valid function. You can safely drop those scenarios.